src/main/java/training/subsystems/Elbow/ElbowSubsystem.java

```diff
@@ -0,0 +1,76 @@
+package training.subsystems.Elbow;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkPIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import utils.GBSubsystem;
+
+import static training.subsystems.Elbow.Constants.*;
+
+public class ElbowSubsystem extends GBSubsystem {
+    private static ElbowSubsystem instance;
+
+    private final CANSparkMax motor = new CANSparkMax(ELBOW_ID, CANSparkLowLevel.MotorType.kBrushless);
+    private final SparkPIDController motorPIDF = motor.getPIDController();
+    private Rotation2d angle = DEFAULT_POSITION_ELBOW;
```

Comment:
assign the values in the constructor (put this. when you assign) , also i dont think you should save the pid controller separately but its fine

Reply:
Should I do the same also for `motorPIDF` and `motor`, and make them non-final, nor should I do that only for changeable vars?

Reply:
you can keep them finals, if you init them in the constructor